src/color-gamut/README.md

```diff
@@ -7,42 +7,20 @@ Gamut indicator. Used internally by `<color-swatch>`
 
 Static (only read once):
 ```html
-<color-gamut>red</color-gamut>
+<color-gamut color="red"></color-gamut>
 ```
 
-Produces <color-gamut>red</color-gamut>
+Invalid color:
+```html
+<color-gamut color="poop"></color-gamut>
+```
 
 Dynamic:
-```js
-colorGamutElement.color = colorValue;
+```html
+<color-gamut id="cg_1" color="red"></color-gamut>
+<script>cg_1.color = "oklch(50% 0.5 180)";</script>
```

Comment:
```suggestion
<script>setTimeout(() => { cg_1.color = "oklch(50% 0.5 180)" }, 0);</script>
```

Reply:
We are facing a race condition with this demo so that I would propose something like this.

Reply:
Authors should not have to care whether the custom element has been defined or not, props should automatically take care of using any values already set on their properties instead of just overwriting them with the defaults.

However, I tried to make a testcase for this bug and can't reproduce it: https://codepen.io/leaverou/pen/rNgLOEM

Reply:
> Authors should not have to care whether the custom element has been defined or not, props should automatically take care of using any values already set on their properties instead of just overwriting them with the defaults.

Thank you!

🤷‍♂️ Not sure what kind of bug it is (and how to reproduce it), but I can show it:

https://github.com/color-js/elements/assets/9166277/b364c7a9-fb8e-4475-941a-96c4474fac1c

Reply:
Lovely, race conditions! Anyhow, I can't even get the static one to work, we should focus on that before dealing with the dynamic version.

Reply:
Should be fixed now. :)